src/App.js

```diff
@@ -1,7 +1,9 @@
+import Homepage from "pages/homepage/homepage";
 
 function App() {
   return (
     <div className="App">

```

Comment:
Is your Nav component going to be constant across all pages?
If yes then you can try placing it in your app component so it renders for all your pages

Reply:
Will do that thanks.